### DISCUSSION TITLE
GitHub Copilot Data

### DISCUSSION THREAD

**Discussion by sepmon86** [community/community#72674] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am a researcher in the field of Information Systems, I am very interested in doing research on Copilot and getting insight regarding how developers are using Copilot in the open-source software development process, if anyone has insight about how it is possible to access the data usage I would appreciate it.

**Comment by Akash1134** [comment, marked as answer]
Hi @sepmon86 ,

You may be able to gather some insights by conducting surveys or interviews with developers who have used GitHub Copilot in their projects. Additionally, you can analyze public repositories on GitHub that have used Copilot to see how it has been integrated into their code. However, any such data is not readily available on our website. 

I'd recommend that you explore the [Copilot Trust Centre](https://resources.github.com/copilot-trust-center/) for details and compiled resources.

Thanks!

**Reply by sepmon86** [reply]
Thank you! Very helpful!

**Reply by SZahraM** [reply]
Hi @Akash1134,

I am a researcher and I'm interested in accessing the logs that record interactions related to my account. Since this doesn't involve data from other users, I hope there might be a way to obtain this information.

Thanks!

**Reply by davecheney** [reply]
@Software-Security-Student please contact github support who can assist you with that request.

**Reply by SZahraM** [reply]
Thank you, @davecheney, for your suggestion. I've already reached out to them, but it seems that the response time is long.
